### ISSUE TITLE
Uncaught TypeError: Cannot set property 'keyBindings' of undefined

### ISSUE THREAD

**Issue by arturkot** [DavidLGoldberg/jumpy#60]
1. Use Atom 0.206.0
2. Attempt ti use Jumpy (shift + enter).

Perhaps it's something connected with [this PR](https://github.com/atom/atom/pull/6733)?

**Atom Version**: 0.206.0
**System**: Mac OS X 10.10.3
**Thrown From**: [jumpy](https://github.com/DavidLGoldberg/jumpy) package, v2.0.6

### Stack Trace

Uncaught TypeError: Cannot set property 'keyBindings' of undefined

```
At /Users/arturkot/.atom/packages/jumpy/lib/jumpy-view.coffee:131

TypeError: Cannot set property 'keyBindings' of undefined
  at JumpyView.module.exports.JumpyView.turnOffSlowKeys (/Users/arturkot/.atom/packages/jumpy/lib/jumpy-view.coffee:131:9)
  at JumpyView.module.exports.JumpyView.toggle (/Users/arturkot/.atom/packages/jumpy/lib/jumpy-view.coffee:146:10)
  at atom-workspace.commands.add.atom.commands.add.jumpy:toggle (/Users/arturkot/.atom/packages/jumpy/lib/jumpy-view.coffee:44:33)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:238:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:520:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:355:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:177:20)

```

### Commands

```
     -0:37.6.0 snippets:next-tab-stop (atom-text-editor.editor.is-focused)
     -0:37.6.0 snippets:expand (atom-text-editor.editor.is-focused)
     -0:37.6.0 editor:indent (atom-text-editor.editor.is-focused)
     -0:37 editor:newline (atom-text-editor.editor.is-focused)
     -0:36.2.0 core:undo (atom-text-editor.editor.is-focused)
     -0:35.6.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:29.1.0 core:save (atom-text-editor.editor.is-focused.autocomplete-active)
  2x -0:12.4.0 editor:select-to-first-character-of-line (atom-text-editor.editor.is-focused)
  3x -0:12 core:backspace (atom-text-editor.editor.is-focused)
  2x -0:10.8.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:09.8.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:09.4.0 core:paste (atom-text-editor.editor.is-focused)
     -0:05.9.0 core:move-up (atom-text-editor.editor.is-focused)
  2x -0:05.7.0 editor:move-to-first-character-of-line (atom-text-editor.editor.is-focused)
     -0:04.6.0 editor:select-to-end-of-line (atom-text-editor.editor.is-focused)
  2x -0:03.8.0 jumpy:toggle (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "ignoredNames": [
      ".git",
      ".svn",
      ".DS_Store"
    ],
    "disabledPackages": [
      "minimap",
      "webbox-color",
      "preview",
      "remember-session",
      "open-last-project",
      "atom-color-highlight"
    ],
    "themes": [
      "unity-ui",
      "base16-tomorrow-light-theme"
    ]
  },
  "jumpy": {}
}
```

### Installed Packages

```coffee
# User
atom-handlebars, v0.5.0
color-picker, v2.0.4
editorconfig, v1.0.0
jumpy, v2.0.6
language-apache, v1.3.0
language-haml, v0.21.0
linter, v0.12.6
linter-coffeelint, v0.3.1
linter-jshint, v0.1.6
linter-jsxhint, v0.1.2
project-manager, v1.15.8
react, v0.12.0
subword-navigation, v2.7.0
unity-ui, v2.0.10

# Dev
No dev packages
```

**Comment by kankaristo** [comment]
Also started happening for me in 0.206.0.

**Comment by DavidLGoldberg** [comment]
I've noticed several things wrong.  It's a shame, because I had no "deprecated calls" going into this release.  I didn't know / see we could do the 1.0 preview stuff.  I'll definitely do that going forward.  The structure used for keymaps changed from keymap->keymaps was one issue.  The bigger issue was that the overlay really changed.  I started looking at this late last night in the AM.  I have to go to work, but tonight I can take a look, I started a branch called fix_59_*.  This is mostly a dupe of 59 I think.  But I'll leave it open because of the link to the PR (appreciate the issue and the help!)

If anyone (maybe elsewhere in the world) wants to help during *my* daytime feel free!  I'm free tonight, so will be looking at this!  For now you'll probably have to disable the package until hopefully tomorrow.
